lib/vector2/CommonVectors.dm

```diff
@@ -27,6 +29,9 @@ GLOBAL_REAL(Vector2, /Vector2) = new
 	Northwest = new(-sqrt(1/2), sqrt(1/2))
 	Southeast = new(sqrt(1/2), -sqrt(1/2))
 	Southwest = new(-sqrt(1/2), -sqrt(1/2))
+	//DS13 Change - Multiz support.
+	Up = new(1, 1)
```

Comment:
Not 100% on these values, could use some insight here. They're mostly just placeholder.

Reply:
nah it doesn't make sense to have vectors for up and down at all, since they represent movement in the third dimension